mri/tests/test_scripts.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# System import
+import unittest
+import numpy as np
+
+# Package import
+from mri.operators import FFT, WaveletN
+from mri.operators.utils import convert_mask_to_locations
+from mri.reconstructors import SingleChannelReconstructor
+from mri.scripts.gridsearch import launch_grid
+
+# Third party import
+from modopt.math.metrics import ssim
+from modopt.opt.linear import Identity
+from modopt.opt.proximity import SparseThreshold
+
+
+class TestScripts(unittest.TestCase):
+    """ Test the scripts in pysap mri like gridsearch
+    """
+    def setUp(self):
+        """ Setup the test.
+        """
+        self.N = 64
+        self.max_iter = 10
+        self.n_jobs = 2
+        self.num_channels = 2
+
+    def test_gridsearch_single_channel(self):
+        """Test Gridsearch script in mri.scripts for
```

Comment:
@zaccharieramzi I am just testing sanity for codes for now, I am not sure what more we can test with gridsearch, any ideas?

Reply:
Coud we define a prescribed set of values for a given parameter to quickly test the grid search functionality?

Reply:
Hmm you mean we know what is the appropriate `mu` and see if we obtain the right value of `mu`? 
For now, the code is general in the sense that the user gets to define what criteria to use for comparing results, I need to still work on result selection aspect, but surely this can be done

Reply:
Not necessarily, here this idea would be to just test that you can correctly iterate over multiple values for the parameter of interest and return the value that gives the best score, whatever the metric used (SSIM, pSNR, etc).

Reply:
Seems like a good idea, let me do this and update